lib/linguist/heuristics.rb

```diff
@@ -68,6 +68,20 @@ def call(data)
     # Common heuristics
     ObjectiveCRegex = /^\s*(@(interface|class|protocol|property|end|synchronised|selector|implementation)\b|#import\s+.+\.h[">])/
 
+
+    CPPRegexType0 = /^\s*#.*include.*<(stdint|cstdint|string|vector|map|list|array|bitset|queue|stack|forward_list|unordered_map|unordered_set|(i|o|io)stream).*>/
```

Comment:
Erm, to clarify: I meant "whitespace and/or comments" within the actual regex.

[Here's a good example](https://github.com/github/linguist/blob/c8094d3/lib/linguist/strategy/modeline.rb#L4-L93) of what I'm talking about.